index.html

```diff
@@ -5,12 +5,85 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Bangers|Caveat|Courgette|Lobster&display=swap" rel="stylesheet">
     <title>Responsive Cake webpage</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
-    <!-- Add your markup here -->
+ <main>
+   <header>
+    <div class="Logo">
+        <img class="logoup" src="Yusuf Logo.png" alt="logo"/>
+        <p>best home made cake delivered at your door</p>
+     </div>
+     <h3>Add To Cart</h3>
+     <div>
+        <ul>
+            <li><a class="first-menu" href="#">Menu</a>
+                <ul>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">Cakes</a></li>
+                    <li><a href="#">Lesson</a></li>
+                    <li><a href="#">Ordering</a></li>
+                    <li><a href="#">Download recipes</a></li>
+                    <li><a href="#">About</a></li>
+                </ul>
+            </li>
+        </ul>
+    </div>
+  </header>
+<table>
```

Comment:
its a nice idea using table however it does mean the when it comes to scaling to smaller screen widths, the table won't be as responsive. Using flexbox container with child elements means your images can then flow without additional styling to smaller screen widths.